src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { MyStatsComponent, SocietyStatsComponent } from './index';
```

Comment:
Importing from index does make this lose some semantics. No?

Reply:
Makes sense. I'll change this.